packages/react-components/src/organisms/EventMaterials.tsx

```diff
@@ -22,22 +25,30 @@ const EventMaterials: React.FC<EventMaterialsProps> = ({
   meetingMaterials,
 }) => (
   <div css={cardsStyles}>
-    {notes === '' ? (
+    {notes === null ? (
```

Comment:
Was wondering if we should create a new `materials` array, with the actual material type data and title, and then map over it on the render.

Reply:
The additional meeting materials work differently since they have a different type, so I thought mapping to generate three of four then adding a fourth would only make it more confusing